ui-modules/blueprint-composer/app/components/dsl-editor/dsl-editor.template.html

```diff
@@ -62,7 +62,7 @@ <h4 class="list-group-item-heading">
                     {{item.name}}
                 </h4>
                 <p class="list-group-item-entity text-right text-muted">
-                    <img ng-src="{{item.entity.icon}}" alt="{{item.entity.miscData.get('typeName') || (item.entity | entityName) || 'New application'}} logo" class="list-group-item-entity-icon" />
+                    <img ng-if="item.entity.icon" ng-src="{{item.entity.icon}}" class="list-group-item-entity-icon"/>
```

Comment:
@ahgittin, this is a proposal to resolve icon issue in DSL search result.

Reply:
This looks a lot cleaner. I like it.

Reply:
:+1: